### PR TITLE
Remove the deployed dashboard json file to stop resetting the changes…

### PR DESCRIPTION
**Pull request by NipunaMarcus**
… done to existing dashboard

### CODE REVIEW

File: apps/portal/js/dashboard-deployer.js [hunk 1]
```diff
@@ -35,6 +35,8 @@ try {
 
                 log.info("Deploying the dashboard: " + dashboardJson.title);
                 dashboard.create(dashboardJson);
+
```
Comment (sinthuja) [inline apps/portal/js/dashboard-deployer.js]: Remove the unnecessary blank lines throughout the code.

File: apps/portal/js/designer.js [hunk 2]
```diff
@@ -935,6 +935,24 @@ $(function () {
...
+     * Check whether options are available or not
+     * @param optionKeys {Object}
+     * @param options {Object}
+     * @return isAvailable {boolean}
+     * @private
+     * */
+    var isOptionAvailable = function (optionKeys, options) {
+        var isAvailable = false;
+        for (var i = 0; i < optionKeys.length; i++) {
+            if (options[optionKeys[i]].type.toUpperCase() != "HIDDEN") {
```
Comment (sinthuja) [inline apps/portal/js/designer.js]: Use constants here.

File: apps/portal/js/designer.js [hunk 3]
```diff
@@ -2507,20 +2528,20 @@ function toggleCaret(e) {
 $('.sidebar-wrapper').on('hidden.bs.collapse', toggleCaret);
 $('.sidebar-wrapper').on('shown.bs.collapse', toggleCaret);
 
-$('#left-sidebar').on('hidden.sidebar', function(e){
+$('#left-sidebar').on('hidden.sidebar', function (e) {
     $(e.target).find('button[data-target=#left-sidebar-sub]').removeClass('active').attr('aria-expanded', 'false');
```
Comment (sinthuja) [inline apps/portal/js/designer.js]: What is this e? Please add appropriate name for 'e'.

File: apps/portal/js/dashboard-deployer.js (hunk 1 above)
Comment (NipunaMarcus) [inline apps/portal/js/dashboard-deployer.js]: This is fixed with the PR https://github.com/wso2/carbon-dashboards/pull/173

File: apps/portal/js/designer.js (hunk 2 above)
Comment (NipunaMarcus) [inline apps/portal/js/designer.js]: This is fixed with PR https://github.com/wso2/carbon-dashboards/pull/173

File: apps/portal/js/designer.js (hunk 3 above)
Comment (NipunaMarcus) [inline apps/portal/js/designer.js]: This is fixed with PR https://github.com/wso2/carbon-dashboards/pull/173

**Pull request merged**